.github/workflows/test.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
 
+    env:
+      SECRET_KEY: "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
+      DEBUG: "False"
```

Comment:
The `SECRET_KEY` is hardcoded in the workflow file, which is a security risk. It's recommended to use GitHub Secrets to store sensitive information securely. You can reference a secret in your workflow using `${{ secrets.SECRET_KEY }}`.